packages/slice-machine/lib/forms/index.ts

```diff
@@ -59,7 +60,7 @@ export const createInitialValues = (FormFields: {
 export const createValidationSchema = (FormFields: {
   // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-explicit-any
   [fieldKey: string]: any;
-}) => {
+}): Yup.AnyObjectSchema => {
```

Comment:
Still sing yup in some places, I'll jot that down on the technical debt board and we might deal with it another day :)